cli/command/context/remove.go

```diff
@@ -0,0 +1,62 @@
+package context
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+type removeOptions struct {
+	force bool
+}
+
+func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
+	var opts removeOptions
+	cmd := &cobra.Command{
+		Use:     "rm CONTEXT [CONTEXT...]",
+		Aliases: []string{"remove"},
+		Short:   "Remove one or more contexts",
+		Args:    cli.RequiresMinArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runRemove(dockerCli, opts, args)
+		},
+	}
+	cmd.Flags().BoolVarP(&opts.force, "force", "f", false, "Force the removal of a context in use")
+	return cmd
+}
+
+func runRemove(dockerCli command.Cli, opts removeOptions, names []string) error {
+	var errs []string
+	currentCtx := strings.ToLower(dockerCli.CurrentContext())
```

Comment:
Why this ToLower? Do you really want to be able to remove without case sensitive? Note that the `create` command is case sensitive. Well to be fair, it will depend the os (case insensitive on windows). But my point is that it should be homogeneous on every command.

Reply:
Ok my bad it's only for current context... But well the comment is still valid I guess?

Reply:
Good catch, there is inconsistency here. I'll introduce a `normalizeName` func and use it on all subcommands.